### ISSUE TITLE
Tiến độ tiến hành chương Natural Language Processing: Pre-Training

### ISSUE THREAD

**Issue by duythanhvn** [aivivn/d2l-vn#2258]
Cập nhật tiến độ hoàn thành chương **Natural Language Processing: Pre-Training**.
Tiến độ toàn dự án xem **[tại đây](https://github.com/aivivn/d2l-vn/issues/1497)**.

## Giai đoạn 1 `45 parts`
* [x] 14.00 [NLP Index](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+index_vn+label%3A%22status%3A+phase+1%22+label%3A%22chapter%3A+nlp-pre%22) `01 part`
* [x] 14.01. [Word Embedding (word2vec)](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+word2vec_vn+label%3A%22status%3A+phase+1%22+label%3A%22chapter%3A+nlp-pre%22) `05 parts`
* [x] 14.02. [Approximate Training](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+approx-training_vn+label%3A%22status%3A+phase+1%22+label%3A%22chapter%3A+nlp-pre%22) `04 parts`
* [x] 14.03. [The Dataset for Pretraining Word Embedding](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+word-embedding-dataset_vn+label%3A%22status%3A+phase+1%22+label%3A%22chapter%3A+nlp-pre%22) `05 parts`
* [x] 14.04. [Pretraining word2vec](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+word2vec-pretraining_vn+label%3A%22status%3A+phase+1%22+label%3A%22chapter%3A+nlp-pre%22) `04 parts`
* [x] 14.05. [Word Embedding with Global Vectors (GloVe)](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+glove_vn+label%3A%22status%3A+phase+1%22+label%3A%22chapter%3A+nlp-pre%22) `05 parts`
* [x] 14.06. [Subword Embedding](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+subword-embedding_vn+label%3A%22status%3A+phase+1%22+label%3A%22chapter%3A+nlp-pre%22) `04 parts`
* [x] 14.07. [Finding Synonyms and Analogies](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+similarity-analogy_vn+label%3A%22status%3A+phase+1%22+label%3A%22chapter%3A+nlp-pre%22) `03 parts`
* [x] 14.08. [Bidirectional Encoder Representations from Transformers (BERT)](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+bert_vn+label%3A%22status%3A+phase+1%22+label%3A%22chapter%3A+nlp-pre%22) `07 parts`
* [x] 14.09. [The Dataset for Pretraining BERT](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+bert-dataset_vn+label%3A%22status%3A+phase+1%22+label%3A%22chapter%3A+nlp-pre%22) `05 parts`
* [x] 14.10. [Pretraining BERT](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+bert-pretraining_vn+label%3A%22status%3A+phase+1%22+label%3A%22chapter%3A+nlp-pre%22) `02 parts`

## Giai đoạn 2 `19 parts`
* [x] 14.00 [NLP Index](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+index_vn+label%3A%22status%3A+phase+2%22+label%3A%22chapter%3A+nlp-pre%22) `01 part`
* [x] 14.01. [Word Embedding (word2vec)](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+word2vec_vn+label%3A%22status%3A+phase+2%22+label%3A%22chapter%3A+nlp-pre%22) `02 parts`
* [x] 14.02. [Approximate Training](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+approx-training_vn+label%3A%22status%3A+phase+2%22+label%3A%22chapter%3A+nlp-pre%22) `02 parts`
* [x] 14.03. [The Dataset for Pretraining Word Embedding](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+word-embedding-dataset_vn+label%3A%22status%3A+phase+2%22+label%3A%22chapter%3A+nlp-pre%22) `02 parts`
* [x] 14.04. [Pretraining word2vec](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+word2vec-pretraining_vn+label%3A%22status%3A+phase+2%22+label%3A%22chapter%3A+nlp-pre%22) `02 parts`
* [x] 14.05. [Word Embedding with Global Vectors (GloVe)](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+glove_vn+label%3A%22status%3A+phase+2%22+label%3A%22chapter%3A+nlp-pre%22) `02 parts`
* [x] 14.06. [Subword Embedding](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+subword-embedding_vn+label%3A%22status%3A+phase+2%22+label%3A%22chapter%3A+nlp-pre%22) `02 parts`
* [x] 14.07. [Finding Synonyms and Analogies](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+similarity-analogy_vn+label%3A%22status%3A+phase+2%22+label%3A%22chapter%3A+nlp-pre%22) `01 part`
* [x] 14.08. [Bidirectional Encoder Representations from Transformers (BERT)](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+bert_vn+label%3A%22status%3A+phase+2%22+label%3A%22chapter%3A+nlp-pre%22) `02 parts`
* [x] 14.09. [The Dataset for Pretraining BERT](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+bert-dataset_vn+label%3A%22status%3A+phase+2%22+label%3A%22chapter%3A+nlp-pre%22) `02 parts`
* [x] 14.10. [Pretraining BERT](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+bert-pretraining_vn+label%3A%22status%3A+phase+2%22+label%3A%22chapter%3A+nlp-pre%22) `01 part`
